### DISCUSSION TITLE
problems with booting nfs-root

### DISCUSSION THREAD

**Discussion by Thomas-D-de** [ltsp/ltsp#797] [Q&A] [answered]
![Bildschirmfoto vom 2023-02-15 15-38-05](https://user-images.githubusercontent.com/87693113/219058349-c38401f6-3614-4b73-b884-f37d94b0ed29.png)

Hi all,
when booting with root from Image over NFS everything is fine.

When trying to boot with NFS-Root: 55-initrd-bottom.sh fails because "init" is not there.

 re mv "$rootmnt/sbin/init" "$rootmnt/sbin/init.ltsp" 

tried with different kernels 5.15.0, 5.15.0-25, 5.15.0-60, 6.1.0-1006 (Ubuntu Jammy)

any ideas?

**Comment by alkisg** [comment, marked as answer]
I think overlayfs over NFS without squashfs broke sometime in the 5.x kernels era.
Try with a much older kernel.
I don't think we can do something from the LTSP side, this need to be reported in the kernel.

**Comment by alkisg** [comment]
@Thomas-D-de does this PR solve the issue?
https://github.com/ltsp/ltsp/pull/859
